lib/tilt/template.rb

```diff
@@ -293,11 +306,11 @@ def compile_template_method(local_keys, scope_class=nil)
       method_source << source
       method_source << "\nend;end;"
 
-      bind_compiled_method(method_source, offset, scope_class, local_keys)
+      bind_compiled_method(method_source, offset, scope_class)
       unbind_compiled_method(method_name)
     end
 
-    def bind_compiled_method(method_source, offset, scope_class, local_keys)
```

Comment:
I noticed this was receiving `local_keys` but doing nothing with it, so I removed this param while I was here.

Reply:
This stems from earlier iterations of the `compiled_path` support, where the local keys were used in determining the final compiled path.  I suppose it is fine to remove.